operator/build.gradle.kts

```diff
@@ -31,10 +31,17 @@ dependencies {
     implementation(libs.jackson)
     implementation(libs.javaoperatorsdk)
 
+    implementation("commons-cli:commons-cli:1.5.0")
```

Comment:
nit: can we follow the `libs.` convention in the other dependencies? see the top level `settings.gradle.kts` file

especially for common stuff like `org.apache.commons`